src/views/ModelPlan/TaskList/_components/TaskListSideNav/index.tsx

```diff
@@ -103,6 +124,18 @@ const TaskListSideNav = ({
               <span aria-hidden /> indexTwo
             </Trans>
           </Button>
+          <Button
+            type="button"
+            onClick={() =>
+              window.open('/help-and-knowledge/sample-model-plan', '_blank')
+            }
+            className="usa-button usa-button--unstyled line-height-body-5"
+          >
+            <Trans i18nKey="modelPlanTaskList:sideNav.sampleModelPlan">
+              indexZero
+              <span aria-hidden /> indexTwo
+            </Trans>
+          </Button>
```

Comment:
I think link could use a larger margin from the link above per Figma.  As is, it appears the two links are one.